src/ExtEventLoop.php

```diff
@@ -178,8 +178,7 @@ public function tick()
 
         $this->futureTickQueue->tick();
 
-        // @-suppression: https://github.com/reactphp/react/pull/234#discussion-diff-7759616R226
-        @$this->eventBase->loop(EventBase::LOOP_ONCE | EventBase::LOOP_NONBLOCK);
+        $this->eventBase->loop(EventBase::LOOP_ONCE | EventBase::LOOP_NONBLOCK);
```

Comment:
Alright I was able to reproduce the issue.

First of all I was slightly wrong, I was going by memory after all: using `trigger_error()` doesn't produce any additional warning, it's only exceptions thrown inside callbacks registered in the loop that trigger the following warning `PHP Warning:  EventBase::loop(): An error occurred while invoking the callback in /home/adaniele/Development/react/vendor/react/event-loop/src/ExtEventLoop.php on line 205`.

Aside from that, I wasn't initially able to reproduce it because it seems to working as expected using PHP 7.x where no additional warnings are emitted, on the other hand when using PHP 5.x these warnings are indeed emitted. This sounded kind of strange so I inspected the source code of the extension and found the culprit: errors when invoking callbacks are handled differently in [`php5/classes/event.c`](https://bitbucket.org/osmanov/pecl-event/src/41be9821b69ce996d59b750ecdbd9c07dffe192b/php7/classes/event.c?fileviewer=file-view-default#event.c-84:90) compared to [`php7/classes/event.c`](https://bitbucket.org/osmanov/pecl-event/src/41be9821b69ce996d59b750ecdbd9c07dffe192b/php5/classes/event.c?fileviewer=file-view-default#event.c-76:82), and I think that the originally intended behaviour is more akin to the one implemented for the PHP 7.x variant. I guess it's worth to ask about this to the original developer just to make sure, personally I think that the PHP 5.x variant should be fixed to implement the same logic of the PHP 7.x one.

Anyway, I also encountered a weird behaviour. Consider the following snippet:

```php
$loop = new React\EventLoop\ExtEventLoop();

$errors = array();
set_error_handler(function ($errno, $errstr) use (&$errors) {
    $errors[] = $errstr;
});

$loop->addTimer(0.001, function () {
    throw new Exception("Exception inside my callback");
});

try {
  $loop->run();
} catch (Exception $ex) {
  restore_error_handler();
}

var_dump($errors);
```

The output is:

```
array(0) {
}
```

Notices aren't printed as expected but... wait `$errors` is empty 🤔  On the other hand, running the same snippet with `set_error_handler()` commented produces the following output:

```
PHP Warning:  EventBase::loop(): An error occurred while invoking the callback in /home/adaniele/Development/react/vendor/react/event-loop/src/ExtEventLoop.php on line 205
array(0) {
}
```

😮